src/Tools/Avalonia.PreviewHost/Avalonia.PreviewHost.csproj

```diff
@@ -5,8 +5,9 @@
         <Nullable>enable</Nullable>
         <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
         <ApplicationManifest>app.manifest</ApplicationManifest>
+        <IsPackable>false</IsPackable>
```

Comment:
May be we should re-arrange build.props if we have more than two setup for assemblies

Reply:
I have moved Avalonia.PreviewHost and Consolonia.Designer into an Experimental folder and added comment stating that these are marked as not packable because they are still experimental.